.gitignore

```diff
@@ -52,6 +52,9 @@ notes.txt
 .metamaskrc
 .metamaskprodrc
 
+# Customized manifest configuration
+.manifest-overrides.json
```

Comment:
I would suggest ignoring `.manifest-overrides*.json`

Because remember we want people to be able to create a few of these and toggle between them, plus be able to put them in folders, and gitignore all of these.

Reply:
Great suggestions ! it makes sense now for different env for example, addressed f2cee6a805235a0c7d4eb9161de8c5c93b98bcbd